docs/docs/how-to/performance/improving-site-performance.md

```diff
@@ -0,0 +1,203 @@
+---
+title: Improving Site Performance
+---
+
+To paraphrase Tolstoy, all fast websites are alike, but all slow websites are slow in different ways.
+
+This document is a brief guide to improve your Gatsby site's performance.
+
+## Part 1: Creating a process for performance improvements
+
+Because websites are hard to make fast and easy to make slow, performance improvement isn't just something that takes place at one point in time. Instead, performance is a system that you put in place to maintain and extend performance gains over time.
+
+### First, choose a testing tool
+
+In order to get a baseline, you need to decide what you consider the "source of truth".
+
+Lighthouse / PageSpeed Insights and Webpagetest are the most common website performance testing tools in the Gatsby community. Lighthouse tends to be seen as more "canonical". Webpagetest tends to be seen as more precise.
+
+These tools measure what are known as "Core Web Vitals", which measure both time to page load and time to page interactivity. [Google's official page](https://web.dev/vitals/) has more detail on what these metrics are.
+
+### Second, set up performance monitoring 
+
+Site performance monitoring becomes increasingly important the more long-lasting your project is. You can put a lot of effort into performance work -- but then ship a performance regression that wipes out all your hard work!
+
+There are two options for this: 
+
+- Gatsby Cloud has Lighthouse performance reports built into its CI/CD. Every time someone opens a pull request or merges into master, a Lighthouse report will be run and the performance score displayed.
+
+- Use a third-party service that offers performance monitoring. Different services offer different options -- scheduled runs on a daily basis or CI/CD integration through Github
+
+For additional precision, run Lighthouse multiple times and take the median result. 
+
+### Third, quantify the impact of each change you make 
+
+While you're doing performance-improvement work, it's important to understand the impact of each change or set of changes. You may find that one change gives you a 2-point Lighthouse improvement and another gives you a 20-point improvement.
+
+ There's an easy way to measure this:
+
+- First, write a pull request with each change or set of changes, using a service like Gatsby Cloud or Netlify that generates deploy previews per-PR.
+- Second, run Lighthouse tests against the deploy preview for that branch and the deploy preview for master. Do not compare the live site to a deploy preview since the CDN setup may differ. 
+- Third, calculate the difference in Lighthouse scores that each script
+
```

Comment:
```suggestion
1. Write a pull request with each change or set of changes, using a service like Gatsby Cloud or Netlify that generates deploy previews per-PR.
2. Run Lighthouse tests against the deploy preview for that branch and the deploy preview for master. Do not compare the live site to a deploy preview since the CDN setup may differ. 
3. Calculate the difference in Lighthouse scores that each script

```